### PR TITLE
failOnWarning="true"

### PR DESCRIPTION
**Pull request by DeepDiver1975**

### CODE REVIEW

File: phpunit.xml.dist [hunk 1]
```diff
@@ -2,6 +2,9 @@
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
          bootstrap="vendor/autoload.php"
+         strict="true"
+	 verbose="true"
+	 failOnWarning="true"
```
Comment (PVince81) [inline phpunit.xml.dist]: fix the indents everywhere ?

**Comment by DeepDiver1975** [comment]
I tend to close this pr - no ci in place in here .... to much effort ....

**Comment by tboerger** [comment]
Enabling the ci is a task of minutes ;)

**Comment by DeepDiver1975** [comment]
> Enabling the ci is a task of minutes ;)

sure - but I expect the developer of a component to write the drone.yml .... I already have enough repos to take care of

@settermjd feel like taking over? THX

**Review by settermjd — COMMENTED** [review]
Thanks.

**Pull request merged**